akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.stream.impl
+package akka.util
```

Comment:
Is it really worth breaking comp over this at all, having them in two places (one deprecated) is two allocs instead of one, hardly something that will swamp the JVM...